modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -152,10 +151,9 @@ public void recordingTestThatShouldHaveCorrectDuration() throws IOException, Int
                     .start();
                 String ffmpegOutput = container.getLogs();
 
-                assertTrue(
-                    "Duration is incorrect in:\n " + ffmpegOutput,
-                    ffmpegOutput.contains("Duration: 00:") && !(ffmpegOutput.contains("Duration: 00:00:00.00"))
-                );
+                assertThat(ffmpegOutput.contains("Duration: 00:") && !(ffmpegOutput.contains("Duration: 00:00:00.00")))
+                    .as("Duration is incorrect in:\n " + ffmpegOutput)
```

Comment:
Not sure, should contain the expected duration?

Reply:
I think that would make the description more accurate

Reply:
See my suggestion below. Feel free to accept, resolve this conversation and then merge 😇